integration-cli/docker_utils.go

```diff
@@ -1564,6 +1564,7 @@ func setupRegistry(c *check.C) *testRegistryV2 {
 		if err = reg.Ping(); err == nil {
 			break
 		}
+		c.Log(err)
```

Comment:
Do we want this inside the loop, or only if failed after 50 attempts (I.e., putting it after the loop)?